1-js/09-classes/01-class/article.md

```diff
@@ -167,79 +167,80 @@ Although, there are important differences.
     alert(User); // class User { ... }
     ```
 
-2. Class methods are non-enumerable.
-    A class definition sets `enumerable` flag to `false` for all methods in the `"prototype"`.
+2. Metodele claselor sunt non-enumerabile.
+    Definiția unei clase setează fanionul `enumerable` ca fiind `false` pentru toate metodele din `"prototype"`.
 
-    That's good, because if we `for..in` over an object, we usually don't want its class methods.
+    Asta este foarte bine, pentru că daca am folosi `for..in` peste un obiect, nu am vrea toate metodele clasei acestuia.
 
-3. Classes always `use strict`.
-    All code inside the class construct is automatically in strict mode.
+3. Clasele folosesc întotdeauna `use strict`.
+    Tot codul din interiorul constructorului clasei este în mod automat in stric mode.
+    Tot codul din cadrul constructorului clasei este automatc in strict mode.
 
-Besides, `class` syntax brings many other features that we'll explore later.
+În plus, sintaxa `class` aduce mult mai multe caracteristici pe care o să le explorăm în curând.
 
-## Class Expression
+## Expresia Clasei
 
-Just like functions, classes can be defined inside another expression, passed around, returned, assigned etc.
+Exact ca și funcțiile, clasele pot să fie definite în interiorul alte expresii, pot să fie trimise dintr-o parte în alta, returnare, alocate etc.
```

Comment:
Grammar suggestion : "in interiorul alte expresii" -> "in interiorul altor expresii"

Reply:
Grammar suggestion : "returnare" -> "returnate"
Grammar suggestion : "alocate" -> "atribuite"